WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeConfirmPaymentViewModel.swift

```diff
@@ -47,10 +54,29 @@ final class BlazeConfirmPaymentViewModel: ObservableObject {
             }
         } catch {
             DDLogError("⛔️ Error fetching payment info for Blaze campaign creation: \(error)")
-            shouldDisplayErrorAlert = true
+            shouldDisplayPaymentErrorAlert = true
         }
         isFetchingPaymentInfo = false
     }
+
+    @MainActor
+    func confirmPaymentDetails() async {
```

Comment:
Opinion: What do you think about renaming this into `createCampaign` as this method actually starts the campaign creation process?

Reply:
I renamed this to `submitCampaign` to match the CTA in a30baf248c3933865772b3dd1dbaad7c9512df6e.